sparse_ho/algo/implicit.py

```diff
@@ -35,29 +35,29 @@ def compute_beta_grad(
         Parameters
         ----------
         X: array-like, shape (n_samples, n_features)
-            design matrix
+            Design matrix.
         y: ndarray, shape (n_samples,)
-            observation vector
+            Observation vector.
         log_alpha: float or np.array, shape (n_features,)
-            logarithm of hyperparameter
+            Logarithm of hyperparameter.
         model:  instance of ``sparse_ho.base.BaseModel``
-            a model that follows the sparse_ho API
+            A model that follows the sparse_ho API.
         get_v: callable
-            function which return the values of the the vector v
+            Function which return the values of the the vector v.
         mask0: ndarray, shape (n_features,)
-            boolean of active active for warm start
+            Boolean of active active for warm start.
         dense0: ndarray, shape (n_features,)
-            initial value of the value on the support for warm start
+            Initial value of the value on the support for warm start.
         quantity_to_warm_start: ndarray
```

Comment:
can we think of a shorter name ? I'm confused, I thought the quantity to arm start was given as mask0, dense0

Reply:
This is an additional quantity to warm start, specific to each method: forward, backward or implicit, hence the generic name `quantity_to_warm_start`